lib/WasmLoader/CantLoadWasm.jsx

```diff
@@ -4,43 +4,45 @@ import Button from '../../components/Shared/Button'
 import { BigTitle, Title } from '../../components/Shared/Typography'
 import { IconGlif } from '../../components/Shared/Icons'
 
```

Comment:
This looks like an ugly diff, but again, this was just from running lint --fix and this should be equivalent. I think it's just from fixing indenting that it made the diff look crazy.